zebra-state/src/service.rs

```diff
@@ -979,6 +979,29 @@ impl Service<ReadRequest> for ReadStateService {
                 }
                 .boxed()
             }
+
+            // For the get_address_tx_ids RPC.
+            ReadRequest::TransactionsByAddresses(_address, _start, _end) => {
+                metrics::counter!(
+                    "state.requests",
+                    1,
+                    "service" => "read_state",
+                    "type" => "transactionsbyaddresses",
```

Comment:
Optional: it might be a bit more readable as

```suggestion
                    "type" => "transactions_by_addresses",
```

Reply:
https://github.com/ZcashFoundation/zebra/pull/4062/commits/b1a69676ca75fed87863b4b6cadcb07225159d34